org.freedesktop.Platform.VulkanLayer.gamescope.yml

```diff
@@ -0,0 +1,39 @@
+id: org.freedesktop.Platform.VulkanLayer.gamescope
+branch: '22.08'
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+separate-locales: false
+
+build-options:
+  cflags: -I/usr/lib/extensions/vulkan/gamescope/include
+  cxxflags: -I/usr/lib/extensions/vulkan/gamescope/include
+  ldflags: -L/usr/lib/extensions/vulkan/gamescope/lib
+  prefix: /usr/lib/extensions/vulkan/gamescope
+  libdir: /usr/lib/extensions/vulkan/gamescope/lib
+  prepend-path: /usr/lib/extensions/vulkan/gamescope/bin
+  prepend-ld-library-path: /usr/lib/extensions/vulkan/gamescope/lib
+  prepend-pkg-config-path: /usr/lib/extensions/vulkan/gamescope/lib/pkgconfig
+  strip: true
+
+modules:
+  - modules/libevdev.yml
+  - modules/libinput.yml
+  - modules/libseat.yml
+  - modules/xwayland.yml
+  - modules/wlroots.yml
+  - modules/gamescope.yml
```

Comment:
splitting everything into modules like that make things hard to read / review / maintain.

Reply:
I intentionally made it module-based so that applications that might ship libraries that conflict with gamescope's can cherry pick things. I don't have plans to change that, especially since it's not a small worry.

Reply:
I'm willing to turn a blind eye to it, but at least the main module should be part of the main manifest.

Reply:
I'm not sure that can be handled like I want? Ideally a third party would add this repo as a submodule, and include what manifests they want. 

If gamescope were in the main manifest, they'd have to built it themselves, configuring everything; considering the hell I went through with gamescope, I wouldn't wish that on anyone. 

This is a pretty unorthodox package, there are probably going to be tons of weird bits like this.

Reply:
Every time we make an exception for an unorthodox package, it ends up being used to get another one like this accepted. I strongly believe people interested in bundling gamescope will figure out copying 50-something lines of YAML.

Reply:
Fair enough, I'll look at merging the two later. Feel free to review the other bits.